site/docs/reference/working-logs-stats.md

```diff
@@ -9,10 +9,20 @@ Access to statistics is still a work in progress. For now, this documentation de
 
 ## Accessing logs
 
-You can access logs by materializing them to an external endpoint, or from the command line.
+You can access logs in the web application, by materializing them to an external endpoint, or from the command line.
+
+### Accessing logs in the web application
+
+The Flow web application displays logs for each running catalog task.
+Navigate to the **Captures** and **Materializations** tabs and select the task of interest.
 
 ### Accessing logs from the command line
 
+:::info Beta
```

Comment:
Just a clarification that the new CLI doesn't yet have a `logs` subcommand, or any other way to show logs.

Reply:
Right -- I was assuming that the plan is to add it at some point (relatively) near-term. Is that right?